### ISSUE TITLE
Doesn't work with ivy

### ISSUE THREAD

**Issue by magnattic** [arkon/ng-inline-svg#111]
When I compile and run my application with the new Ivy rendering engine, I get the following error:

```
Uncaught ReferenceError: Injectable is not defined
    at vendor.js:133116
    at Object../node_modules/ng-inline-svg/__ivy_ngcc__/lib/inline-svg.service.js (vendor.js:133119)
    at __webpack_require__ (runtime.js:791)
    at fn (runtime.js:151)
    at Object../node_modules/ng-inline-svg/__ivy_ngcc__/lib/inline-svg.component.js (vendor.js:132735)
    at __webpack_require__ (runtime.js:791)
    at fn (runtime.js:151)
    at Object../node_modules/ng-inline-svg/__ivy_ngcc__/lib/inline-svg.directive.js (vendor.js:132820)
    at __webpack_require__ (runtime.js:791)
    at fn (runtime.js:151)
```

Versions:
``` 
    "ng-inline-svg": "8.4.1",

Angular CLI: 8.1.0-beta.2
Node: 11.14.0
OS: linux x64
Angular: 8.1.0-next.3
... animations, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... platform-server, router

Package                                    Version
--------------------------------------------------------------------
@angular-devkit/architect                  0.801.0-beta.2
@angular-devkit/build-angular              0.801.0-beta.2
@angular-devkit/build-ng-packagr           0.801.0-beta.2
@angular-devkit/build-optimizer            0.801.0-beta.2
@angular-devkit/build-webpack              0.801.0-beta.2
@angular-devkit/core                       8.1.0-beta.2
@angular-devkit/schematics                 8.1.0-beta.2
@angular/cdk                               8.0.1
@angular/cli                               8.1.0-beta.2
@angular/material                          8.0.1
@ngtools/json-schema                       1.1.0
@ngtools/webpack                           8.1.0-beta.2
@nguniversal/express-engine                8.0.0-rc.1
@nguniversal/module-map-ngfactory-loader   8.0.0-rc.1
@schematics/angular                        8.1.0-beta.2
@schematics/update                         0.801.0-beta.2
ng-packagr                                 5.3.0
rxjs                                       6.5.2
typescript                                 3.5.2
webpack                                    4.34.0
```

**Comment by magnattic** [comment]
Potential fix might be to add this to the services:
```
@Injectable({ providedIn: 'root' })
``` 

It's possible that the `providedIn` syntax is necessary for Ivy, but I am not 100% sure.

**Comment by arkon** [comment]
I tried switching to that in v8.5.0. Could you give it a try?

**Comment by magnattic** [comment]
Thanks for the quick update! Unfortunately the issue persists, I'm not sure what's going on. Maybe it's an issue with my project, can anybody confirm that ng-inline-svg works in a sample app compiled with ivy?

**Comment by arkon** [comment]
It probably doesn't, my other libraries apparently don't work with Ivy either and they all have very similar project setups.

Ideally I'd avoid using Angular CLI (which there is an open PR for), but I honestly haven't spent time looking into it since I've stopped using Angular.

**Comment by arkon** [comment]
Supposedly fixed with Angular 8.2.0-next.0

https://github.com/angular/angular/issues/29564

**Comment by magnattic** [comment]
Unfortunately not. This seems to be the current issue now for the problems: https://github.com/angular/angular/issues/31394
